mobile/lib/entities/album.entity.dart

```diff
@@ -121,8 +121,7 @@ class Album {
         owner.value == other.owner.value &&
         thumbnail.value == other.thumbnail.value &&
         sharedUsers.length == other.sharedUsers.length &&
-        assets.length == other.assets.length &&
-        sortOrder == other.sortOrder;
```

Comment:
Why is this property removed?

Reply:
Without that property, when the album sort ordered is changed from the web, it will not be updated on the mobile app, correct?

Reply:
? This just removes it from the equality comparison.

Reply:
nvm, the condition to pull an update of the album is from the `updatedAt` property. Which seems not updating when the sort order is updated on the server